### PR TITLE
updated hardware info

### PR DESCRIPTION
**Pull request by Sangreal8**

### CODE REVIEW

File: docs/hardware/pygmalion.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 ## Details
 
+* Type: Dell PowerEdge R710
 * OS: Ubuntu 16.04
+* CPU: Intel (R) Xeon (R) E5620 2.46GHz
```
Comment (CSmartt) [inline docs/hardware/pygmalion.md]: 2x of these

File: docs/hardware/pygmalion.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
...
+* RAM: 16GB
+* Network: QLogic bnx2 Gigabit Ethernet Driver v2.2.6
```
Comment (CSmartt) [inline docs/hardware/pygmalion.md]: `01:00.0 Ethernet controller: Broadcom Corporation NetXtreme II BCM5709 Gigabit Ethernet (rev 20)
01:00.1 Ethernet controller: Broadcom Corporation NetXtreme II BCM5709 Gigabit Ethernet (rev 20)
02:00.0 Ethernet controller: Broadcom Corporation NetXtreme II BCM5709 Gigabit Ethernet (rev 20)
02:00.1 Ethernet controller: Broadcom Corporation NetXtreme II BCM5709 Gigabit Ethernet (rev 20)`
Pulled from `lspci` on Pygmalion, we're not using a QLogic NIC.

**Review by CSmartt — CHANGES_REQUESTED** [review]

File: docs/hardware/pygmalion.md (hunk 2 above)
Comment (CSmartt) [reply docs/hardware/pygmalion.md]: Also, 4 of these ^^^

Comment (Sangreal8) [reply docs/hardware/pygmalion.md]: Thanks! :)

**Review by CSmartt — APPROVED** [review]

**Pull request merged**